### DISCUSSION TITLE
New to teslamate

### DISCUSSION THREAD

**Discussion by fjgaspar** [teslamate-org/teslamate#2499] [Getting started] [answered]
I have installed Teslamate in an Ubuntu machine in my home network. I have logged into the webpage usin API tokens, and I have some issues/questions:

First, the main page of my Teslamate, shows a map where my car is parked, and at the right, current status of the car. That works perfectly.

But the dashboards configured (I don't know how it will be named in english, something like Control Panels or so, a number of links appear, each one opens a new tab with graphana panels. Well, if I go to the first, charge level, it opens a panel where the current charge level of the car appears. And thats all. All other panels appears with No data. Except timeline, which shows a software update, but no drive or charging information.

So my guess is that, for data to be collected, it is needed that Teslamate software runs 24x7. I mean, it is not possible to keep it not running, use the car all day, and when at home, run the software and it downloads the data from the car. If that is not the case, I mean, it is possible to keep the Teslamate application off and collect the historical info from the car when starts, please, tell me how to do it. If instead, the app has to be running all the time to be able to store all data, please, let me know as well.

And sorry if the question is too basic, please.

Thanks

**Comment by cwanja** [comment, marked as answer]
Correct, you need to have TeslaMate running either every time you drive, charge, update, etc. OR leave it running 24/7. The application is meant to poll the Tesla owners API live. And save the data in real time. There is no way to dump the data after a drive or a charge, etc.

**Reply by fjgaspar** [reply]
Thankyou. I supposed it was like that. Today I have begun to have data.

**Comment by ktsenvolt** [comment]
Whenever you are using or driving your car you can turn on your home server. Or you can install Teslamate on Raspberry Pi and run it 24/7.

**Reply by fjgaspar** [reply]
Thanks. By the moment, I won't expend money in a Raspberry. Will run in the Ubuntu virtual machine.
